shiny/ui/_layout_columns.py

```diff
@@ -15,19 +15,16 @@
 T = TypeVar("T")
 
 
-class Breakpoints(Enum):
-    """
-    References
-    ----------
-    * [Available Bootstrap breakpoints](https://getbootstrap.com/docs/5.3/layout/breakpoints/#available-breakpoints)
-    """
+Breakpoints = Literal["xs", "sm", "md", "lg", "xl", "xxl"]
+"""
+References
+----------
+* [Available Bootstrap breakpoints](https://getbootstrap.com/docs/5.3/layout/breakpoints/#available-breakpoints)
+"""
+
 
-    xs = "xs"
-    sm = "sm"
-    md = "md"
-    lg = "lg"
-    xl = "xl"
-    xxl = "xxl"
+def bs_breakpoints() -> Iterable[Breakpoints]:
```

Comment:
I can't tell if this is for public use. If so, can we do without the `bs_` prefix? If not, maybe just make this a variable instead of a function.

Reply:
It's not for public use, so I made it a variable and removed the prefix.